resources/assets/components/Post/index.js

```diff
@@ -60,39 +60,43 @@ class Post extends React.Component {
     const signup = this.props.signup;
     const campaign = this.props.campaign;
     const quantity = post.quantity != null ? post.quantity : 0;
+    const containerSize = post.type === 'photo' ? '-third' : '-half';
+    const textOrCaption = post.type === 'photo' ? 'Photo Caption' : 'Text';
```

Comment:
This is a `TODO` for later, but it would be cool if we can return a standard property for this value in the JSON response for a post so the post can decide how the `text` field should be described instead of us having to hard code it. Related to this ticket: https://www.pivotaltracker.com/story/show/154744799

Reply:
oh yes I love that!